### ISSUE TITLE
Unable to run frontend localhost with npm run dev

### ISSUE THREAD

**Issue by marcelobbfonseca** [sbkwgh/forum#14]
Hello, im having troubles trying to run the frontend in localhost.

after running `npm install`  and then `npm run dev` in  './frontend/' i get:

`cosmiconfig requires at least version 4 of Node, please upgrade
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@1.0.0 dev: `node build/dev-server.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the frontend@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcelo/.npm/_logs/2018-12-29T18_11_23_344Z-debug.log`

Im using node 11.2.0
npm 6.4.1

**Comment by sbkwgh** [comment]
Could you provide the log file? There's a lot of things it could be without seeing more info

**Comment by sbkwgh** [comment]
Not sure really? Looks like its some problem with vue's dev-server

Did you definitely do npm install _in_ the frontend directory?

**Comment by marcelobbfonseca** [comment]
Yes, searching around I've found and tried this solution pointed in here https://github.com/vuejs-templates/webpack/issues/1229 but still no success. The solution pointed there also will give npm vulnerability warnings. 
I've also been facing dependencies warning with npm after npm install.
I could try to update the frontend project to vue-cli3 to see if it runs without any problems.

**Comment by marcelobbfonseca** [comment]
it worked! Had to make a few changes to fix some issues. Theres just some features missing without the backend but i could make a pull request or send you the code if you want

**Comment by sbkwgh** [comment]
Amazing! Might take a few days for me to look at it but would be great if you could

**Comment by marcelobbfonseca** [comment]
added here https://github.com/marcelobbfonseca/forum 
I've added it to a vue-cli3 setup
It still need need to be tested with backend and in prod

**Comment by sastava007** [comment]
I have forked **forum** from @marcelobbfonseca with a modified version of frontend code, so I'm able to run on the frontend. But when I'm trying to run backend, it's all running fine but no action can be performed. Getting this weird error
![image](https://user-images.githubusercontent.com/43990819/70390204-f285fe80-19ee-11ea-9b9f-3a16ec6a4315.png)
@sbkwgh Can you please look into this issue?

**Comment by marcelobbfonseca** [comment]
hi @sastava007 sorry for the late reply. what port your backend is running in localhost? are you using docker?
